src/scss/components/_select.scss

```diff
@@ -21,31 +23,83 @@
 // Styling for the choicejs widget for the select component
 // Overwriting styles found here: https://github.com/Choices-js/Choices/blob/master/src/styles/choices.scss
 .#{prefix(form-control--select)} {
+  $control-height: $grid-row-height;
+  $list-hpadding: $grid-margin-2;
+  $list-vpadding: $grid-margin-1;
+  $list-vpaddings:  $list-vpadding * 2;
+  $item-margin: $grid-margin-0/2;
+  $item-margins: $item-margin * 2;
+  $item-hpadding: $grid-margin-1;
+  $vborders: $typography-size-border;
+
   width: 100%;
 
+  // The wrapper div.
+  // Style as a regular input.
   .choices {
     @include body;
     @include border(all, $size: 1px);
-    @include rows(1);
-    appearance: none;
+    @include rows(1, min-height);
+    align-items: center;
     border-radius: 0;
+    display: flex;
```

Comment:
can you please keep `display: flex` and `align-items` grouped together, as they both apply to the flexbox layout

now it's harder to read how the layout is defined.